src/lib/client.ts

```diff
@@ -159,8 +155,7 @@ export class Client implements IClient {
   }
 
   /* FARMER INTEGRATION */
-  // TODO: method should return Promise<void>, update after backend is updated
-  public async startFarming(path: string, plotSizeGB: number): Promise<boolean> {
+  public async startFarming(path: string, plotSizeGB: number): Promise<void> {
```

Comment:
Same here, also could start introducing [tsdoc](https://tsdoc.org/) documentation as you mentioned prior